feature/contributor/src/main/res/layout/fragment_contributor.xml

```diff
@@ -8,7 +8,6 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@drawable/bg_bottom_sheet_dialog_fragment"
```

Comment:
This is not necessary because window background and list background colors are same.

<img width="471" alt="20190513_055922_Pixel 3" src="https://user-images.githubusercontent.com/1269214/57587713-448fa000-7544-11e9-8193-d0c9a6fb6a18.png">